dfastmi/gui.py

```diff
@@ -350,16 +350,19 @@ def update_qvalues() -> None:
     ireach = dialog["reach"].currentIndex()
     if ireach < 0:
         return
+    
+    if rivers.hydro_q is None:
```

Comment:
should there be a message in this case?

Reply:
Removed from branch

Reply:
https://issuetracker.deltares.nl/browse/DFAST-115